packages/datadog_session_replay/ios/datadog_session_replay/Sources/Swift/Feature/FlutterSeasionReplayFeature.swift

```diff
@@ -5,7 +5,17 @@
 import Foundation
```

Comment:
nit: spelling mistake in the name of this file. Should be SessionReplay and says SeasionReplay